website/pages/docs/plugin-portal/index.mdx

```diff
@@ -54,6 +54,7 @@ exists within the Vault repository, the plugin can be built as instructed in
 <Columns count={2}>
 
 - [Cassandra](/api-docs/secret/databases/cassandra)
+- [Couchbase](/api-docs/secret/databases/couchbase)
```

Comment:
```suggestion
- [Couchbase](https://github.com/hashicorp/vault-plugin-database-couchbase)
```
We should use the plugin repo link since that 's available.

Reply:
Sounds good.  Will make note of it for the next time.